docs/visualization/visualize_unit_metrics.ipynb

```diff
@@ -107,14 +107,14 @@
      "name": "stderr",
```

Comment:
<p>You should quantify Firing rate in "binning intervals" so the firing rate would be the total number of spike in a given interval (for example 100 ms) divided by that interval.</p><p> </p><p>Is the code missing below?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AllenInstitute/openscope_databook/pull/139/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
<p>We discussed this, I retract my comment</p>